newIDE/app/src/Utils/GDevelopServices/InAppTutorial.js

```diff
@@ -3,38 +3,73 @@
 import axios from 'axios';
 import { GDevelopAssetApi } from './ApiConfigs';
 import { type InAppTutorial } from '../../InAppTutorial/InAppTutorialContext';
-import VersionMetadata from '../../Version/VersionMetadata';
 import optionalRequire from '../OptionalRequire';
 import Window from '../Window';
-const electron = optionalRequire('electron');
+const fs = optionalRequire('fs').promises;
+const path = optionalRequire('path');
+const remote = optionalRequire('@electron/remote');
+const app = remote ? remote.app : null;
 
 export type InAppTutorialShortHeader = {|
   id: string,
   contentUrl: string,
   availableLocales: Array<string>,
 |};
 
+const readJSONFile = async (filepath: string): Promise<Object> => {
+  if (!fs) throw new Error('Filesystem is not supported.');
+
+  try {
+    const data = await fs.readFile(filepath, { encoding: 'utf8' });
+    const dataObject = JSON.parse(data);
+    return dataObject;
+  } catch (ex) {
+    throw new Error(filepath + ' is a corrupted/malformed file.');
+  }
+};
+
+const fetchVersionnedLocalFileIfDesktop = async (
+  filename: string
+): Promise<?Object> => {
+  const shouldFetchVersionnedTutorials = !!remote && !Window.isDev();
+  if (!shouldFetchVersionnedTutorials) return null;
+
+  const appPath = app ? app.getAppPath() : process.cwd();
+  // If on desktop released version, find json in resources.
+  // This allows making it available offline, and also to fix a version of the
+  // tutorials (so that it's not broken by a new version of GDevelop).
+  const filePath = path.join(
+    appPath,
+    '..', // If on dev env, replace with '../../app/resources' to test.
+    `in-app-tutorials/${filename}.json`
+  );
+  const data = await readJSONFile(filePath);
+  return data;
+};
+
 export const fetchInAppTutorialShortHeaders = async (): Promise<
   Array<InAppTutorialShortHeader>
 > => {
-  const shouldFetchVersionnedTutorials = !!electron && !Window.isDev();
+  const inAppTutorialShortHeadersStoredLocally = await fetchVersionnedLocalFileIfDesktop(
+    'in-app-tutorial-short-header'
+  );
+  if (inAppTutorialShortHeadersStoredLocally)
+    return inAppTutorialShortHeadersStoredLocally;
 
   const response = await axios.get(
-    `${GDevelopAssetApi.baseUrl}/in-app-tutorial-short-header`,
-    {
-      params: {
-        gdevelopVersion: shouldFetchVersionnedTutorials
-          ? VersionMetadata.version
-          : undefined,
-      },
-    }
+    `${GDevelopAssetApi.baseUrl}/in-app-tutorial-short-header`
   );
   return response.data;
 };
 
 export const fetchInAppTutorial = async (
   shortHeader: InAppTutorialShortHeader
 ): Promise<InAppTutorial> => {
+  const inAppTutorialStoredLocally = await fetchVersionnedLocalFileIfDesktop(
+    shortHeader.id
+  );
+  if (inAppTutorialStoredLocally) return inAppTutorialStoredLocally;
```

Comment:
Same, try/catch this and in case of an exception, log it and continue, so that we fallback to the API.